dip-0020.md

```diff
@@ -0,0 +1,221 @@
+<pre>
+DIP: 0020
+Title: Activating old and implementing new opcodes
```

Comment:
@mangus I think we could go with this simplified title as the abstract describes the details effectively. Would this be okay with you? Either way we will need to use the same title in the Readme as in the DIP itself.
```suggestion
Title: Dash Core Opcode Updates
```

Reply:
> Would this be okay with you?

Sure.

Reply:
I'd like to see this changed to "Dash Opcode Updates" or something similar. Since this is a protocol level change, we probably shouldn't say "Dash Core" since that is just one implementation.

Reply:
> I'd like to see this changed to "Dash Opcode Updates" or something similar. Since this is a protocol level change, we probably shouldn't say "Dash Core" since that is just one implementation.

Good point @PastaPastaPasta :+1: Let's update it to what you suggested.